softwareComponents/shapeReconfig/src/hashing.cpp

```diff
@@ -0,0 +1,98 @@
+#include <shapeReconfig/hashing.hpp>
+
+namespace rofi::shapereconfig {
+
+using namespace rofi::configuration;
+
+size_t combineHash( size_t h1, size_t h2 ) 
+{ 
+    return h1 ^ (h2 << 1);
+}
+
+// If converted to degrees and rounded, equal degree has same hash
+size_t hashRad( float rads )
```

Comment:
This is still not ideal - since the degree could change depending on the float calculation, but you can probably leave it.

Reply:
The point I am making is that it doesn't have to be hashed equally...
But there is not much you can do (other than abandon hashing...)